graphql-demo-product/README.md

```diff
@@ -0,0 +1,25 @@
+# GraphQL Demo
+
+This Demo provides a GraphQL endpoint running with the Ivy Engine.
+The endpoint supports querying tasks and users and allows creating new users.
+For more information, see the [GraphQL](https://community.axonivy.com/d/526-graphql) community post.
+
+## Demo
+
+For the following demonstration of sending queries to the endpoint, the [Altair GraphQL Client](https://altairgraphql.dev) is used.
+This client is available for most common platforms.
+Any other client that can send POST requests can be used instead.
+
```

Comment:
were onboarding a new product onto the Market @andreasbalsiger . 
I think for the sake of simplicity we keep the readme as it is now.
However, feel free to suggest or improve the words of promotion later on.